shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -566,11 +575,12 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
         out.write(buf, 0, t);
         t = is.read(buf);
       }
-      if (!tmpDst.renameTo(dstFile)) {
+      if (!tmpDst.renameTo(outputFile)) {
         throw new IOException(
-            "Failed to rename " + tmpDst.getPath() + " to destination " + dstPath);
+            "Failed to rename " + tmpDst.getPath() + " to destination "
+                    + "file://" + outputFile.getPath());
       }
-      System.out.println("Copied " + srcPath + " to " + dstPath);
+      System.out.println("Copied " + srcPath + " to " + "file://" + outputFile.getPath());
```

Comment:
I see, so we would either say "Copied file to /path/to/dir" or "Copied file to /path/to/dir/file". I prefer the former since the user specified "dir", not "dir/file", but I'll leave the final decision up to you